libs/components/docs-tools/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@skyux/docs-tools",
+  "version": "0.0.0-PLACEHOLDER",
+  "author": "Blackbaud, Inc.",
+  "keywords": [
+    "blackbaud",
+    "skyux"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blackbaud/skyux.git"
+  },
+  "bugs": {
+    "url": "https://github.com/blackbaud/skyux/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux#readme",
+  "peerDependencies": {
+    "@angular/common": "^19.0.5",
+    "@angular/core": "^19.0.5",
+    "@angular/platform-browser": "^19.0.5",
+    "@skyux/assets": "0.0.0-PLACEHOLDER",
+    "@skyux/core": "0.0.0-PLACEHOLDER",
+    "@skyux/i18n": "0.0.0-PLACEHOLDER",
+    "@skyux/icon": "0.0.0-PLACEHOLDER",
+    "@skyux/layout": "0.0.0-PLACEHOLDER",
+    "@skyux/tabs": "0.0.0-PLACEHOLDER",
+    "@skyux/toast": "0.0.0-PLACEHOLDER"
+  },
+  "dependencies": {
+    "@stackblitz/sdk": "1.11.0",
+    "highlight.js": "11.11.1",
```

Comment:
`highlight.js` is lightweight and works at runtime. Angular Material docs uses this for their code examples, too.